_format/1.1/index.md

```diff
@@ -1237,7 +1237,8 @@ section of the [compound document].
 
 The value of the `include` parameter **MUST** be a comma-separated (U+002C
 COMMA, ",") list of relationship paths. A relationship path is a dot-separated
-(U+002E FULL-STOP, ".") list of [relationship][relationships] names.
+(U+002E FULL-STOP, ".") list of [relationship][relationships] names. An empty
+value indicates that no related resource should be returned.
```

Comment:
If we wish to add this clarification. 

```suggestion
(U+002E FULL-STOP, ".") list of [relationship][relationships] names. An empty
value (no characters) indicates that no related resource should be returned.
```

Reply:
@prathe I'm not sure the clarification (`no characters`) is needed, but I'm not opposed either. If we were to add this, it should also be added to the similar language around [sparse field sets](https://jsonapi.org/format/1.1/#fetching-sparse-fieldsets). Since it affects multiple areas, maybe a small separate PR would be best.